server/config/topic_page/equity/USA.textproto

```diff
@@ -120,6 +120,26 @@ blocks {
   }
 }
 
+blocks {
+  title: "Income inequality between sexes"
```

Comment:
maybe fine, but I do wonder if we less controversial to just say between "men and women" instead of "sexes"? (given gender identity questions)

Reply:
yea, we do have it both ways. but the census uses "sex" as biological sex where as gender is a social construct (and they are [updating to ask both](https://www.census.gov/library/stories/2021/08/household-pulse-survey-updates-sex-question-now-asks-sexual-orientation-and-gender-identity.html))

how about we add a link to this in the description: https://www.census.gov/topics/population/age-and-sex/about.html

Reply:
adding that reference sounds great!  we sort of have a modeling bug then, in that we map it to "gender" property.

Reply:
ah yes, we should clarify it there (and in this case, update the name of the stat var?)